mainnet/singularity/cluster_config_bootnodes.json

```diff
@@ -27,6 +27,7 @@
     "GENESIS_TOKEN": "QKC",
     "ENABLE_TX_TIMESTAMP": 1561791600,
     "ENABLE_EVM_TIMESTAMP": 1569567600,
+    "ENABLE_QKCHASHX_HEIGHT": 1600000,
```

Comment:
according to @jyouyj this will be around 55 days from now. is this expected? @qizhou please confirm

Reply:
I would suggest early Oct. or about 30 days after now.

Reply:
I suggest "ENABLE_QKCHASHX_HEIGHT": 1400000. Since there are about 8640 minor blocks per day, the estimated enabled time will be Oct. 9th.